### PR TITLE
yank version 0.12.15 of NCDatasets

### PR DESCRIPTION
**Pull request by Alexander-Barth**
This version of NCDatasets allows for netCDF 4.9.2 which has some unresolved issue as described here:
https://github.com/Unidata/netcdf-c/issues/2705

It is not yet clear what is the best fix.

### CODE REVIEW

File: N/NCDatasets/Versions.toml [hunk 1]
```diff
@@ -135,3 +135,4 @@ git-tree-sha1 = "afd015e81e60cfbdba04ef59bcdc80e18bd613cd"
 
 ["0.12.15"]
 git-tree-sha1 = "6450a225f79309a21397e83563287978d5f875a1"
+yanked = true
```
Comment (giordano) [inline N/NCDatasets/Versions.toml:138]:
```suggestion
yanked = true

```

**Comment by giordano** [comment]
This has also a merge conflict.

**Comment by Alexander-Barth** [comment]
I tried to resolve the conflict. But if this issue still persists, we can close the PR, as an version where this issue has been corrected has been released.

Thanks!

File: N/NCDatasets/Versions.toml [hunk 2]
```diff
@@ -135,9 +135,11 @@ git-tree-sha1 = "afd015e81e60cfbdba04ef59bcdc80e18bd613cd"
...
 
 ["0.12.16"]
 git-tree-sha1 = "e201ed836f4486d0a5f593e68b7621b2e24237c5"
 
 ["0.12.17"]
 git-tree-sha1 = "4263c4220f22e20729329838bf7e94a49d1ac32f"
+
```
Comment (giordano) [inline N/NCDatasets/Versions.toml:145]: Now there's an _extra_ new line at the end of the file, before there was none.  There should be _exactly_ one, no more, no less.
```suggestion
```

**Pull request merged**